src/ctap/mod.rs

```diff
@@ -69,6 +69,8 @@ use crate::api::connection::{HidConnection, SendOrRecvStatus};
 use crate::api::customization::Customization;
 use crate::api::firmware_protection::FirmwareProtection;
 use crate::api::upgrade_storage::UpgradeStorage;
+#[cfg(test)]
+use crate::api::user_presence::UserPresenceResult;
```

Comment:
This could go at the beginning of the `mod test`.

Reply:
Yes!

Done.